src/adagrams.js

```diff
@@ -1,7 +1,122 @@
+import { truncate } from "fs";
+import { getConsoleOutput } from "@jest/console";
+
+const letters = ['A','B','C','D','E','F','G','H',
+'I','J','K','L','M','N','O','P','Q','R','S',
+'T','U','V','W','X','Y','Z'];
+
+const weights = [9, 2, 2, 4, 12, 2, 3, 2, 9, 1, 1, 4,
+2, 6, 8, 2, 1, 6, 4, 6, 4, 2, 2, 1, 2, 1];
+
+const randomLetter = () => {
+  let totalWeigth = 0;
```

Comment:
This logic for drawing a letter works, but it has a problem: you put the letter back in the bag each time, so to speak.